### DISCUSSION TITLE
想问一下收藏的视频列表 /mv/sublist 这个接口能不能分页？

### DISCUSSION THREAD

**Discussion by wood3n** [Binaryify/NeteaseCloudMusicApi#1278] [Q&A] [answered]
我个人试了一下如果不传 limit和offset，那么默认返回24条数据，传 limit 参数可以生效，但是 offset 不对，如果 offset 传 0，则从第一条开始返回，但是当 offset = 1 的时候，是从第二条数据开始返回的🤣，我期望是从第二页返回啊不是从第二条给我截断

**Comment by Binaryify** [comment, marked as answer]
offset是偏移量，不是page

**Reply by wood3n** [reply]
好吧，我一直理解错了，我再看看文档
